courses/fundamentals_of_ada/070_subprograms.rst

```diff
@@ -1669,9 +1355,9 @@ Summary
 Summary
 ---------
 
-* Procedures are abstractions for actions
-* Functions are abstractions for value computations
-* Functions may return composite values
+* :ada:`procedure` are abstractions for actions
```

Comment:
if you're going to use singular, then "is an abstraction ..."

Reply:
Done